components/x-article-save-button/src/ArticleSaveButton.scss

```diff
@@ -0,0 +1,36 @@
+@import 'o-icons/main';
+@import 'o-colors/main';
+@import 'o-typography/main';
+
+$icon-size: 38px;
+
+@mixin background-icon($name, $color) {
+	@include oIconsGetIcon($name, oColorsGetPaletteColor($color), $icon-size, $iconset-version: 1);
+}
+
+.article-save-button {
+	display: inline-block;
+}
+
+.article-save-button__button {
+	@include background-icon(bookmark-outline, #000000);
+	@include oTypographySans($scale: -2);
+	border: 0;
+	padding: ($icon-size - 5px) 0 0 0;
+	width: $icon-size;
+	color: getColor('black');
+
+	&:focus {
```

Comment:
Do we need `:active` as well?

Reply:
There doesn't appear to a be an active state for the existing save button.